.circleci/config.yml

```diff
@@ -170,7 +170,8 @@ jobs:
             make build-cmd BUILD_NUMBER='${CIRCLE_BRANCH}-${CIRCLE_BUILD_NUM}'
             make package BUILD_NUMBER='${CIRCLE_BRANCH}-${CIRCLE_BUILD_NUM}'
             curl -sL https://sentry.io/get-cli/ | bash
-            export VERSION=$(sentry-cli releases propose-version)
+            export VERSION=`git rev-parse HEAD`
+            echo Sentry will use version "$VERSION"
```

Comment:
To be safe, I wouldn't export the variable to prevent the env space of forked processes from being polluted. We could just directly use the result of the command in the echo line itself.